scripts/plots/hurricane/exevs_hurricane_regional_tropcyc_plots.sh

```diff
@@ -74,8 +74,10 @@ export PLOTDATA=${STORMroot}
 #export RUN="tropcyc"
 export img_quality="low"
 export fhr_list="0,6,12,18,24,30,36,42,48,54,60,66,72,78,84,90,96,102,108,114,120,126"
-export model_tmp_atcf_name_list="MD01,MD02,MD03,MD04,MD05,MD06"
-export model_plot_name_list="HFSA,HFSB,HWRF,HMON,GFS,CTCX"
+export model_tmp_atcf_name_list="MD01,MD02,MD03,MD04,MD05"
+export model_plot_name_list="HFSA,HFSB,HWRF,HMON,GFS"
+#export model_tmp_atcf_name_list="MD01,MD02,MD03,MD04,MD05,MD06"
+#export model_plot_name_list="HFSA,HFSB,HWRF,HMON,GFS,CTCX"
```

Comment:
@OliviaOstwald-NOAA Can you please add a description before the commented out lines in this section? I recall that we can't use CTCX for 2024 because CTCX is missing the first few storms and the code breaks? Can you add a comment that explains this for @malloryprow and future users? I think we discussed this with @AndrewBenjamin-NOAA, too. Thank you!